src/installer/tests/TestUtils/Command.cs

```diff
@@ -196,7 +197,22 @@ public Command Start()
 
             ReportExecBegin();
 
-            Process.Start();
+            // Retry if we hit ETXTBSY due to Linux race
+            // https://github.com/dotnet/runtime/issues/58964
+            for (int i = 0; i < 3; i++)
+            {
+                try
+                {
+                    Process.Start();
+                    break;
+                }
+                catch (Win32Exception e) when (e.Message.Contains("Text file busy"))
+                {
+                    // 10 ms is short, but the race we're trying to avoid is in-between
+                    // "fork" and "exec", so it should be fast
```

Comment:
to balance latency and likelihood of waiting long enough, perhaps we should scale the backoff a little, like this could be `Sleep(i * 10)` ?

Reply:
It's probably worth seeing if this solves the issue, but there's no reason we can't come back to it later.